Assets/PlayroomKit/PlayroomKit.cs

```diff
@@ -1035,14 +1038,21 @@ private static void InvokeRpcRegisterCallBack(string dataJson, string senderJson
             {
                 Debug.LogError(ex.Message);
             }
-            RpcRegisterCallback?.Invoke(dataJson, senderJson);
+
+            for (int i = 0; i < Math.Min(rpcRegisteredEvents.Count, rpcCalledEvents.Count); i++)
```

Comment:
`rpcRegisteredEvents.Count, rpcCalledEvents.Count`
Why do we give the chance for this inconsistency between two variables? IF there is inconsistency, why don't we fix that?

Reply:
I don't see how this is leading to an inconsistency, if this is about chances of getting Out of bound errors on Index `math.min` should handle that. The need of having 2 lists is both may hold different order, for example we registered 2 events but only 1 was called.

Reply:
What are the reasons we could go Out Of Bounds? Is it because of what you've mentioned (`example, we registered 2 events but only 1 was called.`)

Reply:
> What are the reasons we could go Out Of Bounds? Is one of them `we registered 2 events but only 1 was called.`

That was the reason but right now we are choosing the min value between "rpcRegisteredEvents.Count, rpcCalledEvents.Count". so that case if fixed